extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -640,35 +640,36 @@ export class AmpStoryInteractive extends AMP.BaseElement {
   /**
    * Triggers changes to component state on response interactive.
    *
-   * @param {!Element} optionEl
-   * @private
+   * @param {number} optionIndex
+   * @param {?Element} optionEl
+   * @protected
    */
-  handleOptionSelection_(optionEl) {
+  handleOptionSelection(optionIndex, optionEl) {
```

Comment:
Seems like we can revert this function back to be `@private` again (and add the underscore)